examples/src/index.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import styles from './app.module.css';
+import App from './App';
+
+ReactDOM.render(
+  <div className={styles.testApp}>
```

Comment:
Could the nesting be reduced? For example, `<Root><Chrome><App /></Chrome></Root>`? Or even better, `<Viewport><App /></Viewport>`.